appveyor.yml

```diff
@@ -173,4 +182,9 @@ on_success:
 
 
 on_finish:
-  - ps: 'if (Test-Path ".\junit-results.xml") { (new-object net.webclient).UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\junit-results.xml)) }'
+  - ps: |
+      (new-object net.webclient).UploadFile(
```

Comment:
That doesn't matter. This can fail for a number of reasons, including the file not existing. In either case it will fail, but the lastexitcode will suppress all errors rather than this particular one.,